task-management/src/main/java/org/exoplatform/task/management/controller/TaskController.java

```diff
@@ -495,7 +495,7 @@ public Response comment(Long taskId, String comment, Long parentCommentId, Secur
     Comment cmt = taskService.addComment(taskId, parentCommentId, currentUser, comment); //Can throw TaskNotFoundException
 
     //TODO:
-    CommentModel model = new CommentModel(cmt, userService.loadUser(cmt.getAuthor()), CommentUtil.formatMention(cmt.getComment(), userService));
+    CommentModel model = new CommentModel(cmt, userService.loadUser(cmt.getAuthor()), CommentUtil.formatMention(cmt.getComment(), lang, userService));
```

Comment:
Here you have the current user, so his locale, no need to modify this to put lang in parameter